doc/data/messages/t/too-few-function-args/good.py

```diff
@@ -0,0 +1,7 @@
+class Fruit:
+    def __init__(self, color, name):
+        self.color = color
+        self.name = name
+
+
+apple = Fruit("red", "apple")
```

Comment:
Suggest keeping the example as parallel as possible with bad.py. You might even consider not using a literal `1` as the first argument in bad.py.
```suggestion
isinstance(1, int)
```

Reply:
Done! Also, changed `bad.py` to `isinstance("foo")` and `good.py` to `isinstance("foo", str)` to avoid using the literal `1`.